### PR TITLE
First assignments

### PR DESCRIPTION
**Pull request by crinela**

### CODE REVIEW

File: Students/Pontinteu Crinela/Assignement1_1/src/assign/pack/Main.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		
+		int sum = 0;
+		int givenNr;
+
+		Scanner in = new Scanner(System.in);
+		System.out.println("Input the number ");
+		givenNr = in.nextInt();
+
+		int endLimit;
+		endLimit = (givenNr - 1) / 3;
```
Comment (georgebejan) [inline Students/Pontinteu Crinela/Assignement1_1/src/assign/pack/Main.java]: Check out [this post](http://math.stackexchange.com/questions/9259/find-the-sum-of-all-the-multiples-of-3-or-5-below-1000) for a more efficient approach.

**Comment by georgebejan** [comment]
How about Fibonacci in `O(logn)`?

**Comment by crinela** [comment]
Is the first assignment more correct ?

**Comment by Alex-Cosma** [comment]
Yes, it's good but what happened with your file?

**Comment by crinela** [comment]
I do not know what happened. May I delete the file and put it back?

**Comment by Alex-Cosma** [comment]
Of course

**Comment by crinela** [comment]
Do i have to do something more?

**Comment by Alex-Cosma** [comment]
Nope, you're good

**Pull request merged**